2017/WG-09-12.md

```diff
@@ -0,0 +1,71 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the September 12th video call of WebAssembly's Working Group
+
+- **Host**: Google Hangouts
+- **Dates**: Tuesday September 12th, 2017
+- **Times**: 9:00am–10:00am Pacific Time
+- **Location**: *same Google Hangouts as previous CG video meetings*
+- **Contact**:
+    - Name: Brad Nelson
+    - Email: bradnelson@google.com
+
+### Registration
+
+None required if you've attended before. Email Brad Nelson to sign up if it's
+your first time. The meeting is open to WG members only (we'll likely broaden
+this in future meetings).
+
+## Logistics
+
+The meeting will be a Google Hangouts call.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking.
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Openness (Brad Nelson)
+       1. Discussion on how open the WG process should be.
+       1. POLL: We comfortable with opening WG meetings to all CG members (for observation).
+       1. POLL: We comfortable opening WG meetings to the public (for observation).
+    1. Scope of v.1 specification (Brad Nelson)
+       1. Discussion on which things should be included in the initial
+          specific, possibilities include:
+          1. [Binary format spec](https://github.com/WebAssembly/spec)
+             1. Core format
+             1. Reference interpreter
+             1. Test suite
+          1. [JavaScript embedding](https://github.com/WebAssembly/design/blob/master/JS.md)
+          1. [Web embedding](https://github.com/WebAssembly/design/blob/master/Web.md)
```

Comment:
Indentation of the above two is off?

Reply:
Meant those two to be at same level with spec. Though I suppose for tests it's mixed in, that what you mean?

Reply:
![screenshot 2017-09-06 at 8 50 15 pm](https://user-images.githubusercontent.com/298127/30145255-050e9ec8-9345-11e7-98bf-be58c69898bd.png)

Test suite is more indented than the following two items.

Reply:
done